### DISCUSSION TITLE
PostUpdateTask does not commit file

### DISCUSSION THREAD

**Discussion by lululaplap** [renovatebot/renovate#13863] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

31.15

### Please select which platform you are using if self-hosting.

Bitbucket Cloud (bitbucket.org)

### If you're self-hosting Renovate, tell us what version of the platform you run.

docker

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I am trying to run the package [bumpver](https://pypi.org/project/bumpver/) in PostUpdateTask in order to bump the version in both pyproject.toml and <package>/__init__.py. I am able to confirm that the command runs sucsessfully and the files are modified. However, the files are not commited and the only changes in the diff are the dependancy updates. 

The problem can be reproduced without bumpver using the config below to create a file instead. Having file filters set to `["**/**"]` should mean that all files which have changed should be included in the commit, from my understanding. 

```
module.exports = {
    "platform": "bitbucket",
    "username": "",
    "password": "",
    "repositories": [""],
    "python": {"registryUrls": [""] },
    "allowedPostUpgradeCommands": [".*"],
    "postUpgradeTasks":{
      "commands": ["echo -e 'hello world!' > myfile.txt"],
      "executionMode": "branch",
      "filefilters": ["**/**"]},
    "renovateFork":true,
    "recreateClosed": true,
    "groupName": "all",}
```

### Relevant debug logs

<details><summary>Logs</summary>

```
 INFO: Repository started (repository=)
       "renovateVersion": "31.15.0"
 INFO: Dependency extraction complete (repository=)
       "baseBranch": "master",
       "stats": {
         "managers": {"setup-cfg": {"fileCount": 1, "depCount": 10}},
         "total": {"fileCount": 1, "depCount": 10}
       }
 INFO: Branch created (repository=, branch=renovate/all)
       "commitSha": ""
 INFO: Branch created (repository=, branch=renovate/major-all)
       "commitSha": ""
 INFO: Repository finished (repository=)
       "durationMs": 45870
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
Is that your exact config? `postUpgradeTasks` should be an array, and you should use `fileFilters` instead of `filefilters` (note the capitalization).

**Reply by rarkins** [reply]
@lululaplap please raise discussions first next time and not bug reports

**Reply by lululaplap** [reply]
Thanks for such a quick reply. Fixing the capatalisation solved the problem, thanks for pointing that out.

**Reply by sigma67** [reply]
@rarkins Has this changed or is the documentation wrong? It's an object there.

https://docs.renovatebot.com/configuration-options/#postupgradetasks

**Reply by rarkins** [reply]
Refer to the current documentation as source of truth
